upstream-tools/pre-replacements.json

```diff
@@ -4893,35 +4865,8 @@
     {
       "old": "-> **NOTE:** The parameter `server_side_encryption_configuration` is deprecated.\nUse the resource [`aws_s3_bucket_server_side_encryption_configuration`](s3_bucket_server_side_encryption_configuration.html) instead.\n"
     },
-    {
```

Comment:
This looks really similar (and in the same place, I think, under the key [`"website/docs/r/s3_bucket.html.markdown"`](https://github.com/pulumi/pulumi-aws/pull/2408/commits/0265ad6c5e1d91afbbd22a2b42cfdfd9a469250a#diff-a1ca436dd43a4302dee354a5c8a023e3828382901e55b4fc89230d7c49feb412R2263)) to a replacement that was added in the previous commit. I'm not clear what this means -- does the text get changed now, and not removed?